ui-src/e2e/components/configurationComponent.e2e-spec.ts

```diff
@@ -180,6 +181,7 @@ describe("configuration component", () => {
   });
 
   it("should check that warning appears if value exceeds maximum", () => {
+    configPO.applicationName.sendKeys(101);
```

Comment:
use this test for numeric fields only.
don't think applicationName field should display an error if it contains 101? did you mean a different field here?